tools/qontract_cli.py

```diff
@@ -1760,6 +1765,62 @@ def app_interface_merge_history(ctx):
     print_output(ctx.obj["options"], merge_queue_data, columns)
 
 
+@get.command(
+    short_help="obtain a list of all resources that are managed "
+    "on a customer cluster via a Hive SelectorSyncSet."
+)
+@click.pass_context
+def selectorsyncset_managed_resources(ctx):
+    vault_settings = get_app_interface_vault_settings()
+    secret_reader = create_secret_reader(use_vault=vault_settings.vault)
+    clusters = get_clusters()
+    oc_map = init_oc_map_from_clusters(
+        clusters=clusters,
+        secret_reader=secret_reader,
+        integration="qontract-cli",
+        thread_pool_size=1,
+        init_api_resources=True,
+    )
+    columns = [
+        "cluster",
+        "SelectorSyncSet_name",
+        "SaaSFile_name",
+        "kind",
+        "namespace",
+        "name",
+    ]
+    data = []
+    for c in clusters:
+        c_name = c.name
+        oc = oc_map.get(c_name)
+        if not oc or isinstance(oc, OCLogMsg):
+            continue
+        if "SelectorSyncSet" not in (oc.api_resources or []):
+            continue
+        selectorsyncsets = oc.get_all("SelectorSyncSet")["items"]
+        for sss in selectorsyncsets:
+            try:
+                for resource in sss["spec"]["resources"]:
+                    kind = resource["kind"]
+                    namespace = resource["metadata"].get("namespace")
+                    name = resource["metadata"]["name"]
+                    item = {
+                        "cluster": c_name,
```

Comment:
so that's the hive cluster, right, not the customer cluster. Would it make sense to add the clusterdeployment label selector from the SelectorSyncSet, so we can see (or guess) which customer clusters this applies to ?

Reply:
it probably would. i'm just here to "make it work". happy to see further contributions to "make it work right" and "make it beautiful" :heart:

Reply:
the reason i didn't add it is because it is usually not required when looking at a resource.